src/OrchardCore/Orchard.Mvc.Core/Extensions/ModularServiceCollectionExtensions.cs

```diff
@@ -62,6 +61,13 @@ public static class ModularServiceCollectionExtensions
             return services;
         }
 
+        public static void AddTagHelpers(this IServiceProvider serviceProvider, string assemblyName)
```

Comment:
Oh yes. I didn't do it because `Orchard.Resources` just reference `O.ResourceManagement.Abstractions`, so i could not use a taghelper type coming from `O.ResourceManagement`.

But you're right, i will add overloads. Maybe also one with a Type `typeof(MyTagHelper)`. And still one with an Assembly `typeof(MyTagHelper).GetTypeInfo().Assembly`. And use one of them in `Orchard.Contents`.